release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -74,6 +74,16 @@ In {product-title} {product-version}, the installation program uses Cluster API
 
 Additionally, SSH access to control plane and compute machines is no longer open to the machine network, but is restricted to the security groups associated with the control plane and compute plane machines.
 
+[id="ocp-4-16-installation-and-update-ingress-capability_{context}"]
+==== Ingress capability
+
+Ingress capability is now a configurable cluster capability and is optional for Red{nbsp}Hat Hypershift. It is not configurable and always enabled for standalone {product-title}.
```

Comment:
```suggestion
Ingress capability is now a configurable cluster capability and is optional for Red{nbsp}Hat Hypershift. It is not configurable and is always enabled for standalone {product-title}.
```

Reply:
@jmanthei   I think we need an _is_ here, or am I being too picky?

Reply:
It's correct either way, but I'm happy to add an "is" if you think it's more clear!